lib/ASN1/Object.php

```diff
@@ -288,6 +288,10 @@ protected static function parseIdentifier($identifierOctet, $expectedIdentifier,
 
     protected static function parseBinaryIdentifier($binaryData, &$offsetIndex)
     {
+        if (strlen($binaryData) <= $offsetIndex) {
+            throw new ParserException('Can not parse binary identifier from data: Offset index larger than input size', $offsetIndex);
```

Comment:
I think the exception message would be a little bit clearer if we just mention `identifier` instead of `binary identifier`. In the end its just binary data - what do you think?